### PR TITLE
variations now works

### PR DESCRIPTION
**Pull request by t0bbl**
small changes to get variations of the choosen design

### CODE REVIEW

File: components/FourPicsTable.js [hunk 1]
```diff
@@ -26,7 +26,10 @@ export default function FourPicsTable() {
     <>
       {shirts.map((shirt) => (
         <FourPicsContainer key={shirt._id}>
-          <StyledA href={`/PreviewPage/${searchID}/${shirt.pic1.picSRCSlug}`}>
+          <StyledA
```
Comment (mbosselmann) [inline components/FourPicsTable.js:29]: `StyledLink` would be a better name as this is less confusing (`A` could mean the character `A` or anchor tag. ;))

File: pages/api/openai/variations/index.js [hunk 2]
```diff
@@ -3,25 +3,41 @@ import Shirt from "@/db/models/Shirt";
 
 export default async function handler(req, res) {
   if (req.method === "POST") {
-    const { picSRC, searchID } = req.body.arg;
-
-    await fetchImages(res, { picSRC, searchID });
+    const { picSRC, searchID, picSRCSlug } = req.body.arg;
+    const bufferToUseWithOpenAI = await handlePicture(picSRC);
+    bufferToUseWithOpenAI.name = "temp.png";
+    console.log(bufferToUseWithOpenAI);
```
Comment (mbosselmann) [inline pages/api/openai/variations/index.js:9]: The console.log is not necessary anymore or? 🤔

File: package.json [hunk 3]
```diff
@@ -14,6 +14,7 @@
     "eslint": "8.39.0",
     "eslint-config-next": "13.3.1",
     "file-saver": "^2.0.5",
+    "jotai": "^2.1.0",
```
Comment (mbosselmann) [inline package.json:17]: Do you use jotai? 🤔  If not I would recommend to remove the package.

**Review by mbosselmann — COMMENTED** [review]
Hi Robin,
LGTM! 🦊 I left some suggestions, but nothing that would hinder me of approving your PR.
Happy hacking! :octocat:

File: components/FourPicsTable.js (hunk 1 above)
Comment (t0bbl) [reply components/FourPicsTable.js:29]: there is a style update upcoming, i try to remember your suggestion!

File: package.json (hunk 3 above)
Comment (t0bbl) [reply package.json:17]: i tried something what didnt work but iam going to implement it later anyway so it just chills here for now...

**Pull request merged**